src/ubuntu/manifest.json

```
@@ -651,6 +651,23 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/ubuntu/22.04/",
+              "os": "linux",
+              "osVersion": "jammy",
+              "tags": {
+                "ubuntu-22.04-arm64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "ubuntu-22.04-arm64$(FloatingTagSuffix)": {},
+                "ubuntu-22.04-arm64-local": {
+                  "isLocal": true
+                }
+              }
```

The `local` tag can be removed. This is only needed if another Dockerfile in this repo needs to reference this as a base image.

Also adding the `variant` field.
```suggestion
              "tags": {
                "ubuntu-22.04-arm64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
                "ubuntu-22.04-arm64$(FloatingTagSuffix)": {}
              },
              "variant": "v8"
```

---

Thanks for explaining! Removed the `local` tag

CI also failed on missing `variant`, added it to the declaration